website/content/docs/plugins/plugin-management.mdx

```diff
@@ -18,6 +18,18 @@ will automatically attempt to determine the `api_addr` as well.
 Detailed information regarding the plugin system can be found in the
 [internals documentation](index.mdx).
 
+## Plugin distribution methods
+
+OpenBao supports two methods for distributing external plugins:
+
+1. **Manual management** - Manual download of plugin binaries
+2. **OCI-based distribution** - Automatic download from container registries
+   with declarative configuration. See the [OCI
+   Plugin](/docs/configuration/oci-plugins) configuration reference for complete
+   documentation.
+
+Both methods require the plugin to be registered in OpenBao before it can be used.
```

Comment:
I kinda feel based on discussion in #1476 as well (https://github.com/openbao/openbao/issues/1476#issuecomment-3229250024 + https://github.com/openbao/openbao/issues/1476#issuecomment-2999543561) that we might as well make it auto-register. I don't think it'd be that hard to do, tbh, and declarative audit devices provide an approach for it (two different `types` to denote sources).